From d8b348b461426ecb8b6193b3a980818f3b53b5e8 Mon Sep 17 00:00:00 2001
From: Michal Marek <mmarek@suse.cz>
Date: Mon, 15 Apr 2013 13:24:06 +0200
Subject: [PATCH 1/2] input: Export matrix_keypad_parse_of_params()
Patch-mainline: Submitted 2013-04-16

Couple of input drivers depend on this symbol:

ERROR: "matrix_keypad_parse_of_params" [drivers/input/keyboard/tca8418_keypad.ko] undefined!
ERROR: "matrix_keypad_parse_of_params" [drivers/input/keyboard/cros_ec_keyb.ko] undefined!

Reported-by: Randy Dunlap <rdunlap@infradead.org>
Acked-by: Dmitry Torokhov <dmitry.torokhov@gmail.com>
Signed-off-by: Michal Marek <mmarek@suse.cz>
---
 drivers/input/matrix-keymap.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/input/matrix-keymap.c b/drivers/input/matrix-keymap.c
index 619b382..08b61f5 100644
--- a/drivers/input/matrix-keymap.c
+++ b/drivers/input/matrix-keymap.c
@@ -68,6 +68,7 @@ int matrix_keypad_parse_of_params(struct device *dev,
 
 	return 0;
 }
+EXPORT_SYMBOL_GPL(matrix_keypad_parse_of_params);
 
 static int matrix_keypad_parse_of_keymap(const char *propname,
 					 unsigned int rows, unsigned int cols,
-- 
1.8.2.1

